### PR TITLE
feature/shorten_group_description

### PR DESCRIPTION
**Pull request by dawit-simeon**
The description was too long for the character limit. Removed the links for the configs that were listed.

### CODE REVIEW

**Review by jeff-simeon — CHANGES_REQUESTED** [review]
i don't think we should list out every applied config here...seems like a recipe to make an error

**Comment by dawit-simeon** [comment]
what should we put as the end user impact in that case?

File: Source/Resources/Content/MSGraph/Groups/Baseline - Microsoft 365 Users.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+  "$description": "| What does this do? | Creates a group that is used to assign configurations that should be applied to all Microsoft 365 users. |\n| Why should you use this? | If you want to apply certain Simeon Baseline configurations to your users. |\n| What is the end-user impact? | Users in this group will have the following configurations applied. |\n| Sign-In Risk Policy, User Risk and MFA Registration Policy, Password Reset Policy, Baseline - Protect Policy Managed Client Apps on Unmanaged Android Devices, Baseline - Protect Policy Managed Client Apps on Unmanaged iOS Devices, Baseline - Configure Policy Managed Client Apps on Unmanaged iOS Devices, Baseline - Microsoft 365 Users - Preview, Device Registration Policy |\n| Learn more | [Assign policies to users and groups](https://docs.microsoft.com/en-us/microsoftteams/assign-policies-users-and-groups) |",
```
Comment (jeff-simeon) [inline Source/Resources/Content/MSGraph/Groups/Baseline - Microsoft 365 Users.json:2]: I would put in a general statement that user policies in the baseline are assigned to this group. Same thing with corporate devices

Comment (dawit-simeon) [reply Source/Resources/Content/MSGraph/Groups/Baseline - Microsoft 365 Users.json]: Made some changes. Take a look.

**Review by david-simeon — APPROVED** [review]

File: Source/Resources/Content/MSGraph/Groups/Baseline - Microsoft 365 Users.json (hunk 1 above)
Comment (david-simeon) [reply Source/Resources/Content/MSGraph/Groups/Baseline - Microsoft 365 Users.json]: Looks good to me

**Review by rdorta-simeon — APPROVED** [review]

**Review by jeff-simeon — APPROVED** [review]

**Review by jeff-simeon — APPROVED** [review]

**Pull request merged**